src/popup/components/IconBoxed.vue

```diff
@@ -1,9 +1,13 @@
 <template>
-  <div class="icon-boxed">
+  <div
+    class="icon-boxed"
+    :class="{ [variant]: variant }"
```

Comment:
Consider this change:
```suggestion
    :class="[ variant ]"
```